tests/dummy/app/models/car.js

```diff
@@ -0,0 +1,12 @@
+import Model from 'ember-data/model';
```

Comment:
(restored comment)  
I've created new model for tests for customizing url to not pollute existing ones